site2/docs/helm-deploy.md

```diff
@@ -350,6 +350,7 @@ helm upgrade --install pulsar apache/pulsar \
     --timeout 10m \
     --set [your configuration options]
 ```
+> For the first deployment, add `--set initialize=true` option to initialize bookie and pulsar cluster meta.
```

Comment:
Two questions:
1: it is a note? if yes, please use this format:
> **Note** 
> For the first deployment, add `--set initialize=true` option to initialize bookie and Pulsar cluster metadata.

2: If this update is only available for master? if not, could you please help update docs in other releases? Thanks

Reply:
1: Yes, it is a note, will update.
2: It is also available for other release, will add PR for other releases.